misc/scripts/install_vulkan_sdk_macos.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Download and install the Vulkan SDK.
+curl -LO "https://sdk.lunarg.com/sdk/download/latest/mac/vulkan-sdk.dmg"
+hdiutil attach vulkan-sdk.dmg -mountpoint /Volumes/vulkan-sdk
+sudo "/Volumes/vulkan-sdk/InstallVulkan.app/Contents/MacOS/InstallVulkan" \
+    --root "$HOME/VulkanSDK" --accept-licenses --default-answer --confirm-command install
```

Comment:
```suggestion
"/Volumes/vulkan-sdk/InstallVulkan.app/Contents/MacOS/InstallVulkan" \
    --accept-licenses --default-answer --confirm-command install
```

`sudo` seems to be not required anymore, and there's no need to specify `--root` since Godot build scrips were updated to detect version.